lib/game.rb

```diff
@@ -3,12 +3,14 @@
 require './lib/board'
 
 class Game
+  attr_reader :user_board, :comp_board
 
   def initialize
-  end
+    @comp_board = Board.new
+    @user_board = Board.new
+    # require "pry"; binding.pry
 
-  @comp_board = Board.new
-  @user_board = Board.new #returning nil, why?
+  end
```

Comment:
Took way too long to figure out and may be redundant? But it works. Which I think is now our motto. Looks bad, but it works.